CONTRIBUTING.md

```
@@ -52,6 +52,100 @@ Once we have all the requirements properly installed and configured, we can buil
+
+```shell
+./mvnw generate-test-sources -pl swatch-tally/ct -Pcomponent-tests -am
+```
+
+### Avoiding Maven / IntelliJ Build Conflicts
+
+Maven and IntelliJ both write annotation processor output (MapStruct impls, Hibernate
+metamodel classes) to `target/generated-sources/annotations/`. If Maven has already generated
+these files and IntelliJ tries to regenerate them, you will get an error like:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)